### DISCUSSION TITLE
Fresh install: Admin Account will not login.

### DISCUSSION THREAD

**Discussion by FatherJasper** [requarks/wiki#3148] [Help / Questions]
Good afternoon!

I installed wiki.js with the following guide: https://docs.requarks.io/install/ubuntu

When I try to login after completion it says the username and password for the admin account does not exist. I reinstalled the OS and tried again as well to be sure it wasn't a mistype.

Any guidance on how to resolve this?

**Comment by mattpr** [comment]
Did you get the setup page after install (https://computingforgeeks.com/wp-content/uploads/2019/12/install-wikijs-centos-8-03-1024x628.png)?  

You should have been able to enter your email/password there.
If the setup page worked and gave you the main screen, then you should be able to login with the same credentials you set.
If not, likely a typo on one page or the other.  Since it is a new install, just delete/recreate the db and run the setup page again.
